boards/common/arduino_due/include/w5100_params.h

```diff
@@ -34,10 +36,10 @@ extern "C" {
 #define W5100_PARAM_SPI_CLK     (SPI_CLK_5MHZ)
 #endif
 #ifndef W5100_PARAM_CS
-#define W5100_PARAM_CS          (GPIO_PIN(2, 29))
+#define W5100_PARAM_CS          (ARDUINO_PIN_10)
```

Comment:
Nearly missed that change above :-)

Reply:
what do you mean with missed? Do you disagree with this change?

Reply:
No forget it, everything 's fine with it